tests/phpunit/class-wc-stripe-webhook-handler-test.php

```
@@ -1319,9 +1319,60 @@ public function test_process_checkout_session_does_not_schedule_metadata_job_on_
+		$this->mock_webhook_handler->method( 'get_intent_from_order' )
+			->willReturn( (object) array_merge( self::MOCK_PAYMENT_INTENT, [ 'payment_method' => null ] ) );
+
+		$this->mock_webhook_handler->method( 'get_latest_charge_from_intent' )
+			->willReturn( (object) self::MOCK_PAYMENT_INTENT['charges']['data'][0] );
+
+		$prop = new ReflectionProperty( WC_Stripe_Webhook_Handler::class, 'action_scheduler_service' );
+		$prop->setAccessible( true );
+		$prop->setValue( $this->mock_webhook_handler, $mock_scheduler );
+
+		$this->mock_webhook_handler->process_webhook( wp_json_encode( $notification ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_process_async_checkout_session_success_for_on_hold_order is not named in camelCase. (undefined)

(CamelCaseMethodName)